### ISSUE TITLE
Set up GitHub Workflow to deploy to Google Cloud Run

### ISSUE THREAD

**Issue by cobraz** [getbranches/todoist-github-bot#5]
We probably want a nice workflow that automatically deploys to Cloud Run at every change/some change.

**Comment by cobraz** [comment]
I've been working on a nice workflow concerning #3. 

Unfortunately, I hit a road block: https://github.com/getbranches/todoist-github-bot/actions/runs/3509906877/jobs/5879351946

**Comment by cobraz** [comment]
I think I fixed the issues in https://github.com/getbranches/todoist-github-bot/commit/0ed0256a81c9f669688b2cba3dbaff516c17b6cc

**Comment by cobraz** [comment]
Another issue: The Cloud Run API is not activated, think I'm going to add that to our Pulumi config (in getbranches/conf)

**Comment by cobraz** [comment]
> … think I'm going to add that to our Pulumi config (in getbranches/conf)

Here: https://github.com/getbranches/conf/commit/bff14104f2f019ca68aa091226479d84297a9852

**Comment by cobraz** [comment]
Waiting for workflow run: https://github.com/getbranches/conf/actions/runs/3527047427

**Comment by cobraz** [comment]
Retrying!

**Comment by cobraz** [comment]
New error, but this one is expected! 🚀

```
Revision 'todoist-github-bot-00001-xof' is not ready and cannot serve traffic. The user-provided container failed to start and listen on the port defined provided by the PORT=8080 environment variable. Logs for this revision might contain more information. Logs URL: https://console.cloud.google.com/logs/viewer?project=todoist-github-bot&resource=cloud_run_revision/service_name/todoist-github-bot/revision_name/todoist-github-bot-00001-xof&advancedFilter=resource.type%3D%22cloud_run_revision%22%0Aresource.labels.service_name%3D%22todoist-github-bot%22%0Aresource.labels.revision_name%3D%22todoist-github-bot-00001-xof%22 For more troubleshooting guidance, see https://cloud.google.com/run/docs/troubleshooting#container-failed-to-start
```

I'm closing this, as this is removed once we fix #4
